db/migrate/20231004214930_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      # t.integer :user_id
+      # t.integer :post_id
```

Comment:
- [ ] [OPTIONAL] Kindly remove the unused code in order to have a clean code base.